### ISSUE TITLE
Accessibility - keyboard access - video player controls

### ISSUE THREAD

**Issue by Adstokoe19** [biati-digital/glightbox#339]
The video player elements are not accessible to keyboard commands and therefore affect accessibility for the end user. 

The issue can be seen at: https://biati-digital.github.io/glightbox/#examples under the 'Video gallery' example. Open the first video and then try to tab through the elements:

<img width="774" alt="image" src="https://user-images.githubusercontent.com/47350188/179735987-0102c7c9-c41b-4d42-8d69-9d4141b8a7c9.png">
 
Only the 'previous', 'next' and 'close' elements can be accessed by keyboard.

**Comment by biati-digital** [comment]
Duplicate of #169
